managing_providers/cloud_providers/_topics/adding_amazon_cloud_regions.md

```diff
@@ -5,9 +5,11 @@ the appliance server. You can use this capability to add new regions that have b
 set up since {{ site.data.product.title_short }} was released.  Once adding the region it will be
 available when creating a new Amazon EC2 provider.
 
-1.  Click ![config gear](../../images/config-gear.png) (**Configuration**).
+1.  Click ![config gear](../../images/config-gear.png) **Settings** > **Application Settings**.
 
-2.  Click on the **Settings** accordion, then click **ManageIQ Region** at the top.
+2.  Click on the **Settings** accordion, then click **{{ site.data.product.title_short }}: Region** at the top.
+    Optionally, a server can be selected (within Zones) to limit the additional
+    region to that server only.
```

Comment:
The `Advanced` tab is available both at the Region and Server level:
![image](https://github.com/ManageIQ/manageiq-documentation/assets/1637291/59a778e2-2768-4a67-a9e0-7717d21f5a5c)